### PR TITLE
Issue 3868 fix

### PR DESCRIPTION
**Pull request by PrakashDurlabhji**

### CODE REVIEW

File: src/projects/components/projectsCard/ProjectCardBody.jsx [hunk 1]
```diff
@@ -44,7 +51,7 @@ function ProjectCardBody({ project, projectCanBeActive, currentMemberRole, descL
         line={descLinesCount}
         truncateText="..."
         text={_.unescape(project.description)}
-        textTruncateChild={showLink ? <Link className="read-more-link" to={showLinkURL || projectDetailsURL}>read more</Link> : <span className="read-more-link">read more</span>}
+        textTruncateChild={showLink ? <span className="read-more-link" onClick={(evt) => { goToProjectDetails(evt, projectDetailsURL) }}>read more</span> : <span className="read-more-link">read more</span>}
```
Comment (maxceem) [inline src/projects/components/projectsCard/ProjectCardBody.jsx:54]: Before we took the link from `showLinkURL` and only after the `projectDetailsURL`. Please, don't remove it as some component might use it.

Comment (PrakashDurlabhji) [reply src/projects/components/projectsCard/ProjectCardBody.jsx]: @maxceem done

**Comment by PrakashDurlabhji** [comment]
@maxceem updating

**Comment by PrakashDurlabhji** [comment]
@maxceem PR updated

**Pull request merged**